site/content/contribute/getting-started/labels.md

```diff
@@ -27,8 +27,8 @@ We leverage [GitHub labels](https://help.github.com/en/articles/about-labels) to
 * `1: PM Review`: Requires review by a [product manager](/contribute/getting-started/core-committers/#product-managers).
```

Comment:
nit: do we need the indices 1,2,3,4 for the PM/UX/.. Review labels (since we don't have them for the other labels in the set) and do we need to duplicate the index for PM and UX.

Reply:
It's actually a sort of hack: this forces them to sort to the top of the list in the GitHub labels screen.